colcon_bundle/verb/_overlay_utilities.py

```diff
@@ -17,61 +18,49 @@
 
 
 def create_workspace_overlay(install_base: str,
-                             workspace_staging_path: str,
+                             ws_staging_path: str,
```

Comment:
changing the api seems unecessary, any reason to do so? (this wouldn't be an API change in e.g. C, but in python you can specify kwargs by name)